brainzutils/__init__.py

```diff
@@ -1,7 +1,7 @@
-from pkg_resources import get_distribution, DistributionNotFound
+from importlib_metadata import version, PackageNotFoundError
```

Comment:
let's add a TODO here saying that we use this package for 3.7 and can remove it when we change to 3.8

There is a way of specifying minimum python versions in setup.py, we should add that and set it to 3.7